_pages/rfc-policies.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: pageminimal
+title: "RFC Policies"
+permalink: /rfc-policies
+---
+
+# RFC Policies and Documents
+
+   * [Code of Conduct](/code-of-conduct)
+   * [Culture and Process](/culture-and-process)
+   * [Enforcement](/enforcement)
+   * [RFC Process](/rfc-process)
+   * [Signup Policy](/rfc-signup)
+   * [Slackbot Language Shorthands](/rfc-slackbot-language-shorthands)
+   * [Pairing & Review](/rfc-pairing)
+   * [Docents](/rfc-docents)
+   * [Mentorship: Mentee Guide](/rfc-mentorship-mentee)
+   * [Mentorship: Mentor Guide](/rfc-mentorship-mentor)
```

Comment:
this is a much better way than nesting URLs to make sure those links don't die 😁